packages/odyssey-react-mui/src/theme/useContrastMode.test.tsx

```diff
@@ -377,19 +377,21 @@ describe("useContrastMode and related functions", () => {
   describe("Color utility functions", () => {
     describe("hexToRgb", () => {
       it("converts black hex to rgb", () => {
-        expect(hexToRgb("#000000")).toBe("rgb(0, 0, 0)");
+        expect(hexToRgb("#000000").asFormattedString).toBe("rgb(0, 0, 0)");
       });
 
       it("converts white hex to rgb", () => {
-        expect(hexToRgb("#ffffff")).toBe("rgb(255, 255, 255)");
+        expect(hexToRgb("#ffffff").asFormattedString).toBe(
+          "rgb(255, 255, 255)",
+        );
       });
 
       it("converts mixed color hex to rgb", () => {
-        expect(hexToRgb("#ff0088")).toBe("rgb(255, 0, 136)");
+        expect(hexToRgb("#ff0088").asFormattedString).toBe("rgb(255, 0, 136)");
       });
 
       it("correctly converts HueNeutral50 token to rgb", () => {
-        const result = hexToRgb(Tokens.HueNeutral50);
+        const result = hexToRgb(Tokens.HueNeutral50).asFormattedString;
```

Comment:
I'd probably name this `.formattedString` because `asFormattedString` makes me think it should be a function call: `.asFormattedString()`. Caught me by surprise it wasn't a function call.